frontend/src/components/LanguageSelector.js

```diff
@@ -10,10 +14,10 @@ import portuguese_img from '../media/portuguese.png'
 import '../styles/navbar.css'
 
 const languages = [
-  { value: 'english', label: 'English (US)', flag: english_img },
-  { value: 'french', label: 'French', flag: french_img },
-  { value: 'spanish', label: 'Spanish', flag: spanish_img },
-  { value: 'portuguese', label: 'Portuguese', flag: portuguese_img }
+  { value: 'English', label: 'English', flag: english_img },
+  { value: 'French', label: 'Français', flag: french_img },
```

Comment:
use in-language names of languages for consistency (label, user-side)
however the application still uses the english names (value, dev-side)